### ISSUE TITLE
Hard to Code

### ISSUE THREAD

**Issue by ghost** [h3rald/vscode-min-lang#2]
It is hard to code via using the extension. There are no snippets that the extension supports.

Snippets should be for: operator signatures, typeclasses, constructors, documentation, multi-line comments etc.

**Comment by h3rald** [comment]
Yup, makes sense, definitely for operators, as they are a bit verbose...

Patches are welcome of course if you know how to add them 😉 -- I gotta look it up, it shouldn't be that hard.
